src/components/layout/navbar-items.tsx

```diff
@@ -33,24 +35,25 @@ export const NavbarItems: FC = () => {
                 variant="filled"
                 className={styles.navLinkHover}
             />
-
-            <Group justify="left" pl="xs" c="white">
-                <OrganizationSwitcher
-                    afterSelectOrganizationUrl="/"
-                    appearance={{
-                        elements: {
-                            organizationSwitcherTrigger: {
-                                color: 'white !important',
-                                '& span': { color: 'white !important' },
-                                padding: 0,
-                                '&:hover': {
-                                    backgroundColor: 'var(--mantine-color-blue-9)',
+            {isMultiple && (
+                <Group justify="left" pl="xs" c="white">
+                    <OrganizationSwitcher
+                        afterSelectOrganizationUrl="/"
+                        appearance={{
+                            elements: {
+                                organizationSwitcherTrigger: {
+                                    color: 'white !important',
+                                    '& span': { color: 'white !important' },
+                                    padding: 0,
+                                    '&:hover': {
+                                        backgroundColor: 'var(--mantine-color-blue-9)',
+                                    },
                                 },
                             },
-                        },
-                    }}
-                />
-            </Group>
+                        }}
+                    />
+                </Group>
+            )}
```

Comment:
I kinda dislike having this much code inside `{}`,  can we make this be a `OrgSwitcher` component that returns null if `isMultiple == false` ?

Reply:
can be inside same file, just would be nice to have outside the main block to tidy it up